functionality_testing.py

```diff
@@ -0,0 +1,101 @@
+import time
```

Comment:
@vimalsheoran I think this file should go inside test folder. You can create a folder and add a readme file for this.